packages/cubejs-teradata-jdbc-driver/src/TeradataQuery.ts

```diff
@@ -0,0 +1,139 @@
+import { BaseFilter, BaseQuery } from '@cubejs-backend/schema-compiler';
+import R, { concat } from 'ramda';
+
+const GRANULARITY_TO_INTERVAL: Record<string, string> = {
+  day: 'DD',
+  hour: 'HH',
+  minute: 'MI',
+  second: 'second',
+  month: 'MONTH',
+  year: 'YEAR'
+};
+
+class TeradataFilter extends BaseFilter {
+  public likeIgnoreCase(column: any, not: any, param: any, type: string) {
+    const p = (!type || type === 'contains' || type === 'ends') ? '%' : '';
+    const s = (!type || type === 'contains' || type === 'starts') ? '%' : '';
+    return `${column}${not ? ' NOT' : ''} LIKE CONCAT('${p}', ${this.allocateParam(param)}, '${s}')`;
+  }
+}
+
+export class TeradataQuery extends BaseQuery {
+  public newFilter(filter: any) {
+    return new TeradataFilter(this, filter);
+  }
+
+  public concatStringsSql(strings: string[]) {
+    return `CONCAT(${strings.join(', ')})`;
+  }
+
+  public convertTz(field: string) {
+    return `${field}`;
+  }
+
+  //  TO CHECK
+  public timeStampCast(value: string) {
+    console.log('timeStamp timeStampCast', value);
```

Comment:
Do we still need all the console.log()'s?
Does it appear in other Cube.js db drivers as well?
Is there a standard logging solution (e.g. winston) for Cube.js code?

Reply:
removed